include/odp/api/spec/traffic_mngr.h

```diff
@@ -611,14 +641,16 @@ void odp_tm_egress_init(odp_tm_egress_t *egress);
  * a SET of capabilities and not just one, is because it is expected that
  * many HW based implementations may have one set of limits for the HW and
  * also support a SW TM implementation with a (presumably larger) different
- * set of limits.  There are also cases where there could be more than
+ * set of limits.  There are also cases where there could be more than one
  * SW implementation (one supporting say tens of thousands of tm_queues and
- * a variant supporting tens of millions of tm_queues).
+ * a variant supporting tens of millions of tm_queues). It returns capabilities
+ * that are valid for all egresses.
```

Comment:
> "It returns capabilities that are valid for all egresses."

There is a faulty pronoun reference here (i.e. it is not clear at all what "it" refers to). The sentence is also a bit hidden in there. I suggest removing it and rephrasing the first sentence (the one that starts "The odp_tm_capabilities()") like this:

This function returns the complete set of TM capabilities for all egresses.

You could also rephrase the corresponding sentence of the description of odp_tm_egress_capabilities() in the same way.